docs/UserGuide.md

```diff
@@ -875,11 +936,11 @@ delivery? No worries! Just edit the details of a delivery simply by using this f
 **Examples:**
 
 - `delivery edit 1 --status CANCELLED --note Sudden overseas business trip to attend to.` </br>
-  Edits Delivery 1's Delivery Status to CANCELLED and edits the delivery's note. (If delivery previously did not
-  have a note, a note will be created to the delivery)
+  Edits Delivery Status of the delivery ,with Delivery ID of 1, to CANCELLED and edits the note of the Delivery.
```

Comment:
Is the spacing around the comma weird? 
Edits Delivery Status of the **D**elivery, with Delivery ID of 1, to CANCELLED...